xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Code/Translations.xml

```diff
@@ -192,6 +192,7 @@ help.videos.videoCard5.topic2=Learn how to use your application
 help.videos.videoCard6.title=Add / Remove Extensions
 help.videos.videoCard6.topic1=Discover how to add/remove extensions using the Extension Manager
 help.videos.videoCard6.topic2=Discover the XWiki Administration UI
+help.videos.hint=The video in this card is an alternative to the documentation behind those links:
```

Comment:
I'm not sure a screen reader user is aware what "this card" is. Maybe rather something like:

```suggestion
help.videos.hint=The following documentation links provide an alternative to the preceeding video:
```

- but even then I'm not sure that it is clear to the user that there was a video, in particular as the iframe is marked with `role="presentation"`. What about putting this information in the title of the iframe and maybe even removing the `role="presentation"`?

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2466/commits/016b9d71aee7a2b9ef98122380e644bbe2aa399c :+1: 

I decided to remove the presentation role, and add the hint as an 'aria-description' to the iframe. This way, it can be read out to AT users and it's clearly concerning the content of this iframe. Updated the L10N value accordingly.